extensions/ingestion/IngestionQueuePopulator.js

```diff
@@ -39,6 +58,83 @@ class IngestionQueuePopulator extends QueuePopulatorExtension {
         this._incrementMetrics(entry.bucket);
     }
 
+    _filterValueOp(entry) {
```

Comment:
This looks like it's a wrapper for some of the logic in the `filter` function, otherwise it looks to me like it's unused.